### ISSUE TITLE
{Linked:Bug41431665; LinkedBug} [Voice access - Adaptive Cards]: More than one number is provided to access the Combo box edit field using show numbers command.

### ISSUE THREAD

**Issue by Susmitha474** [microsoft/AdaptiveCards#7931]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

Test Environment:

OS Build: 25201.1000
Edge dev: Version 107.0.1379.1 (official build) dev (64-bit)

Application URL: [Schema Explorer | Adaptive Cards](https://adaptivecards.io/explorer/)

Accessibility Tool: Voice Access

Pre-requisite:
OS Settings->Accessibility->Voice Access (Turn ON)

Repro Steps:
Launch the above-mentioned URL on edge Dev with Valid credentials.
Command: "Voice access wake up"
Command: "Click Input.Choiceset".
Command: "Expand What color do you want" combo box.
Command: "Shownumbers".
Observe 2no's are given for the single control in the combo box.
Observe the issue.
Actual Result:
more than 1number is provided for the What is your name combo box.

Expected Result:
only 1number should be given for a single control.

Note:
Path: (Action.Execute-->click what is your first name edit field)
Same issue exist for manage profile control.
PFA: more no's for manage profile.mp4.
Same issue exists for Target Version control present in the header.
PFA: Target version extra numbers.mp4

User Experience:
Voice access users will get impact and will get disoriented if this behavior persists.

MAS Reference:
[No Disruption of Accessibility Features - Liquid (microsoft.com)](https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/04.03.01)

Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-chisha%40microsoft.com%7c1334b110f98744a7f5df08d968eb842b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656183104980596%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=r4ndSHk/LrBDwzi7FO%2BjKN0DCnqUSUcELBvGRkgc5CU%3D&reserved=0)"

### Screenshots

https://user-images.githubusercontent.com/104482612/192793619-59a5c570-73d9-446e-a836-07dada3c7668.mp4

https://user-images.githubusercontent.com/104482612/192793646-8e9da06c-2026-4196-8c2f-6edd2c813c21.mp4

https://user-images.githubusercontent.com/104482612/192794582-fa1d8672-047b-43d5-a269-e74bf56282f2.mp4

### Card JSON

```JSON
NA
```

**Comment by anna-dingler** [comment]
Hi @Susmitha474!

I have started a PR to resolve the issue with the "Target version" combo box. Only one element will be focusable once the PR is completed.

For the "What color do you want" combo box, we are utilizing an input HTML element. This issue is also occurring on the [Mozilla site](https://developer.mozilla.org/en-US/docs/web/html/element/select), so I believe this portion of the bug should be filed on the Voice Access team.

The last control (manage profile example) is unrelated to our site. The popup menu you are referencing is a feature implemented on your browser, and we do not have control over it.

Let me know if you have any questions for me - thanks!

**Comment by Susmitha474** [comment]
closing this bug as per the above comments.

**Comment by KarunaNarang** [comment]
As per Partner bug - https://microsoft.visualstudio.com/OS/_workitems/edit/41431665 , This is an issue with the combo boxes in general which report multiple focussable elements (like the drop down arrow, the text field within the box and the overall combobox. Fixing this will be a very specific hard coding which would dont intend to do. 

Hence closing our bug.
